txt2vectorgfx.py

```diff
@@ -99,15 +99,24 @@ def run(self, p, poFormat, poOpaque, poTight, poKeepPnm, poThreshold):
 
     def check_protrace_install(self) -> str:
         # For Linux, run potrace from installed binary
-        if platform == "linux" or platform == "linux2":
+        if platform == "darwin":
+            try:
+                # check whether already in PATH 
+                checkPath = subprocess.Popen(["potrace","-v"])
+                checkPath.wait()
+                return "potrace"
+            except (Exception):
+                raise Exception("Cannot find installed Protrace on Mac. Please run `sudo apt install potrace`")
```

Comment:
be Mac aware here as well please. 
if MAC, then "brew install potrce" ? I dont know exactly

Reply:
I'm sorry, the leftover code w('sudo') was an oversight. You are correct - it's [at Brew](https://formulae.brew.sh/formula/potrace):

        brew install potrace

Reply:
you can update into this PR , right?

Reply:
just change your local file and push it